README.md

```diff
@@ -202,7 +208,7 @@ The graph of the ROS nodes is shown below:
 One can plan a new path by setting a new goal with the *2D Nav Goal* button in rviz. The planned path should show up in rviz and the drone should follow the path, updating it when obstacles are detected. It is also possible to set a goal without using the obstacle avoidance (i.e. the drone will go straight to this goal and potentially collide with obstacles). To do so, set the position with the *2D Pose Estimate* button in rviz.
 
 
-## Simulating stereo-vision
+#### Simulating stereo-vision
```

Comment:
How does stereo vision differ from the techniques we used in previous section? Is it better/worse? Why do we bother? The answer might not need to be here - it might be in the introduction I propose right up the top where we "set the scene" on what is supported and why.